exchanges/gateio/gateio_wrapper.go

```diff
@@ -1537,10 +1548,13 @@ func (g *Gateio) GetActiveOrders(ctx context.Context, req *order.GetOrdersReques
 					continue
 				}
 				var side order.Side
-				side, err = order.StringToOrderSide(spotOrders[x].Orders[x].Type)
+				side, err = order.StringToOrderSide(spotOrders[x].Orders[x].Side)
 				if err != nil {
 					log.Errorf(log.ExchangeSys, "%s %v", g.Name, err)
 				}
+				if req.Side != order.AnySide && req.Side != side {
+					continue
+				}
```

Comment:
Is that a must to provide an order side? if so, the validating call doesn't tolerate unknown side.
I wanted to include orders with side==ANY so that the order side will not be considered to filter.

Reply:
Yes you must add `order.AnySide` or a specific order.Side to the `GetOrdersRequest`. This is for standardization. Please reinstate the validate check as it also checks for other fields. If there is a filtering issue please let me know.

Reply:
```
if req.Side != order.AnySide && req.Side != side {
					continue
				}
```
but I needed this check for filtering the orders since I can't pass the order side for GetSpotOpenOrders() directly.